packages/coinstac-graphql-schema/src/queries.js

```diff
@@ -0,0 +1,33 @@
+const sharedFields = require('./shared-fields');
+
+const queries = {
+  allDockerImages: `
+    {
+      fetchAllComputations {
+        computation {
+          dockerImage
+        }
+      }
+    }
+  `,
+  fetchAllComputationsMetadata: `
```

Comment:
The fields each return are different and used in different places. The Docker query is only fetching dockerImage names for the computation registry and the Metadata query is returning everything. That's not to say we couldn't just return all the metadata to the computation registry query as well, but, since it's not needed, it seemed like overkill.